sunspot/lib/sunspot/search/stats_row.rb

```diff
@@ -10,35 +10,43 @@ def initialize(data, facet = nil, value = nil) #:nodoc:
       end
 
       def min
-        data['min']
+        data['min'].to_i
```

Comment:
do we really need to change existing behaviour? This could be a breaking change for some users